qa/L0_trace/trace_context.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
```

Comment:
@oandreeva-nv do we need a copyright statement for this file?

Reply:
Good catch, will follow up shortly